plugins/modules/k8s_log.py

```diff
@@ -26,36 +26,8 @@
   - Analogous to `kubectl logs` or `oc logs`
 extends_documentation_fragment:
   - community.kubernetes.k8s_auth_options
+  - community.kubernetes.k8s_name_options
 options:
-  api_version:
-    description:
-    - Use to specify the API version. in conjunction with I(kind), I(name), and I(namespace) to identify a
-      specific object.
-    - If using I(label_selector), cannot be overridden
-    default: v1
-    aliases:
-    - api
-    - version
-    type: str
-  kind:
-    description:
-    - Use to specify an object model. Use in conjunction with I(api_version), I(name), and I(namespace) to identify a
-      specific object.
-    - If using I(label_selector), cannot be overridden
-    required: no
-    default: Pod
```

Comment:
Hmm, this one has a different default than the other modules, that's a bit inconvenient